### DISCUSSION TITLE
Cold smoking

### DISCUSSION THREAD

**Discussion by ryanmurray** [nebhead/PiFire#76] [Ideas]
Neb one thing I used to turn up my p setting and maybe smoke plus can already do is try to make my traegar smoke a ton but also keep temps lower than 160 around 100 to 120 to smoke fish. I realize many people use a smoker box for this but I'm curious if a combination of temperature control, reigniting, and longer wait times between auger pushes could result in having the ultimate BBQ smoker. Essentially just keep it smoldering...

**Comment by faisalhusain007** [comment]
I found this Video on youtube [here](https://www.youtube.com/watch?v=o7ZvUBL-5_Y&t=3s)
His findings are interesting

**Reply by nebhead** [reply]
Oh, this is a neat idea.  I think in combination with @faisalhusain007's DC Fan efforts, this could be a possibility.  I wouldn't mind figuring out how to cold smoke if possible.  I wonder if cycling the fan on/off could achieve something similar.  But yeah, if there is a flame out, then kick the ignitor back on during this process would be a good thing to try.  Worth taking a look at.

**Comment by ryanmurray** [comment]
Yeah its looking like a very fine fuel and fan control would have to
happen. Which ofcourse we only have fan on and fan off. one thing i found
digging is the smoke daddy cold smoker hopper and fan that adds on.

On Mon, Jan 31, 2022 at 9:12 AM faisalhusain007 ***@***.***>
wrote:

> I found this Video on youtube here
> <https://www.youtube.com/watch?v=o7ZvUBL-5_Y&t=3s>
> His findings are interesting
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nebhead/PiFire/discussions/76#discussioncomment-2083129>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AALA66FQLRJBRZ73HCD2UPDUY27G7ANCNFSM5NGRV7QQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Ryan Murray

**Comment by ryanmurray** [comment]
Maybe with adjustable fan and a way to auger a certain fuel load in (would
take some fine tuning) a smokey pile of pellets could be made and have the
hot rod cycling to keep the smolder going....

On Mon, Jan 31, 2022 at 10:35 AM Ryan Murray ***@***.***> wrote:

> Yeah its looking like a very fine fuel and fan control would have to
> happen. Which ofcourse we only have fan on and fan off. one thing i found
> digging is the smoke daddy cold smoker hopper and fan that adds on.
>
> On Mon, Jan 31, 2022 at 9:12 AM faisalhusain007 ***@***.***>
> wrote:
>
>> I found this Video on youtube here
>> <https://www.youtube.com/watch?v=o7ZvUBL-5_Y&t=3s>
>> His findings are interesting
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/nebhead/PiFire/discussions/76#discussioncomment-2083129>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AALA66FQLRJBRZ73HCD2UPDUY27G7ANCNFSM5NGRV7QQ>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
> Ryan Murray
>


-- 
Ryan Murray

**Comment by ryanmurray** [comment]
Maybe I'm just over thinking it now that I've done more research. Maybe a better use would be use a cold smoke tube and experiment if running the fan manually and setting the pi fire in monitor mode would be a better way 

https://youtu.be/NoToSJITXlY

As you can see other people have been hacking there fan on where we would not have to :)

And GMG grills implemented an adjustable fan speed fan only mode for cold smoking so I think your DC fan mod may be the missing link 

Ned how would it be to code in a "cold smoke mode" with a simple adjustable fan on time fan off time?

**Reply by faisalhusain007** [reply]
@ryanmurray I think it's doable without too many challenges; we could set a 15%-20% duty cycle for the dc fan and under work cycle in the settings just play with the auger on time  tuned to your grill 

![image](https://user-images.githubusercontent.com/36872476/151940310-3390fb96-d981-429f-868d-1f1e8918d338.png)

might be an option to be able to set custom off times too though if it's not too much work. Need @nebhead to help. 

Could also work with pulsing the standard fan too. Just a thought
